rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -41,7 +41,7 @@ static_assert(sizeof(float16) == 2, "Bad size");
 namespace pybind11 {
 namespace detail {
 constexpr int NPY_FLOAT16 = 23;
-// Kinda following: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
+// Similar to the implementation in: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
```

Comment:
@fiona-gladwin why do we have to mention this at all?

Reply:
Removed the line